sdk/servicebus/service-bus/test/utils/aadUtils.ts

```diff
@@ -1,4 +1,4 @@
-import msRestNodeAuth from "@azure/ms-rest-nodeauth";
+import * as msRestNodeAuth from "@azure/ms-rest-nodeauth";
```

Comment:
This specific change is to fix the following error.
![image](https://user-images.githubusercontent.com/10452642/71699113-f3524c00-2d72-11ea-972f-b5268041fcd7.png)